mobileapps/tests.py

```diff
@@ -0,0 +1,254 @@
+from edx_solutions_api_integration.test_utils import (
+    APIClientMixin,
+)
+from xmodule.modulestore.tests.django_utils import (
+    ModuleStoreTestCase,
+    TEST_DATA_SPLIT_MODULESTORE
+)
+
+
+@ddt.ddt
+class MobileappsApiTests(ModuleStoreTestCase, APIClientMixin):
+    """ Test suite for Mobileapps API views """
+
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
+
+    def setUp(self):
+        super(MobileappsApiTests, self).setUp()
+        self.base_mobileapps_uri = reverse('mobileapps')
+
+        self.test_user = UserFactory()
+
+        self.test_mobileapp_name = str(uuid.uuid4())
+        self.test_mobileapp_identifier = str(uuid.uuid4())
+        self.test_mobileapp_operating_system = 1
+        self.test_mobileapp_current_version = str(uuid.uuid4())
+        self.test_mobileapp_updated_by = self.test_user.id
+
+        cache.clear()
+
+    def setup_test_mobileapp(self, mobileapp_data=None):
+        """
+        Creates a new mobileapp with given mobileapp_data
+        if mobileapp_data is not present it would create mobile app with test values
+        :param mobileapp_data: Dictionary witch each item represents mobile app attribute
+        :return: newly created mobile app
+        """
+        mobileapp_data = mobileapp_data if mobileapp_data else {}
+        data = {
+            'name': mobileapp_data.get('name', self.test_mobileapp_name),
+            'identifier': mobileapp_data.get('identifier', self.test_mobileapp_identifier),
+            'operating_system': mobileapp_data.get('operating_system', self.test_mobileapp_operating_system),
+            'current_version': mobileapp_data.get('current_version', self.test_mobileapp_current_version),
+            'updated_by': mobileapp_data.get('updated_by', self.test_mobileapp_updated_by),
+            'users': mobileapp_data.get('users', []),
+            'organizations': mobileapp_data.get('organizations', []),
+        }
+        response = self.do_post(self.base_mobileapps_uri, data)
+        self.assertEqual(response.status_code, 201)
+        return response.data
+
+    def test_mobileapps_list_get(self):
+        mobile_apps = []
+        organizations = []
+
+        organizations.append(Organization.objects.create(name='ABC Organization'))
+        organizations.append(Organization.objects.create(name='XYZ Organization'))
+
+        users = UserFactory.create_batch(8)
+
+        for i in xrange(30):
+            data = {
+                'name': 'Test Mobile App {}'.format(i),
+                'identifier': 'Test Mobile App Identifier {}'.format(i),
+                'current_version': '1.0',
+                'updated_by': self.test_mobileapp_updated_by,
+                'users': [user.id for user in users],
+                'organizations': [organization.id for organization in organizations],
+            }
+            mobile_apps.append(self.setup_test_mobileapp(mobileapp_data=data))
+
+        test_uri = '{}'.format(self.base_mobileapps_uri)
```

Comment:
`test_uri` does not make sense here. I think we should use `self.base_mobileapps_uri`